xroad-catalog-collector/README.md

```diff
@@ -1,9 +1,31 @@
-# Introduction to X-Road Catalog Collector
-
-The purpose of this module is to collect members, subsystems and services from the X-Road ecosystem and store them to 
-a database. 
-
-The module is implemented using JAVA virtual threads: 
+# X-Road Catalog Collector
+
+## Table of Contents
+
+- [X-Road Catalog Collector](#x-road-catalog-collector)
+    - [Table of Contents](#table-of-contents)
+    - [Introduction to X-Road Catalog Collector](#introduction-to-x-road-catalog-collector)
+    - [Build](#build)
+    - [Configuration](#configuration)
+        - [Mandatory to Provide](#mandatory-to-provide)
+            - [Mandatory Configurations for Data Source and Liquibase](#mandatory-configurations-for-data-source-and-liquibase)
+            - [Mandatory Configurations for Common Features](#mandatory-configurations-for-common-features)
+            - [Mandatory Configurations for Finland-specific Features](#mandatory-configurations-for-finland-specific-features)
+        - [Optional configurations](#optional-configurations)
+            - [Optional Configurations for Common Features](#optional-configurations-for-common-features)
+            - [Optional Configurations for Finland-specific Features](#optional-configurations-for-finland-specific-features)
+        - [Fixed-Mandatory values to include in `application.yaml`](#fixed-mandatory-values-to-include-in-applicationyaml)
+            - [Fixed-Mandatory Values for Data Source and Liquibase](#fixed-mandatory-values-for-data-source-and-liquibase)
+            - [Fixed-Mandatory Values for Spring Boot Framework](#fixed-mandatory-values-for-spring-boot-framework)
+    - [Run](#run)
+    - [Run against a remote Security Server over an SSH tunnel](#run-against-a-remote-security-server-over-an-ssh-tunnel)
```

Comment:
Since we use the asterix (`*`) style of unordered lists in other files, it would be good to use it here as well.